components/pandadoc/pandadoc.app.mjs

```diff
@@ -23,6 +24,26 @@ export default {
         })) || [];
       },
     },
+    completedDocumentId: {
+      type: "string",
+      label: "Completed Document Id",
+      description: "Select a completed document or provide its ID",
+      async options() {
+        const response = await this.listDocuments({
+          params: {
+            deleted: false,
+            status: DOCUMENT_STATUS_COMPLETED,
+          },
+        });
```

Comment:
Should include pagination:
```suggestion
      async options({ page }) {
        const response = await this.listDocuments({
          params: {
            deleted: false,
            status: DOCUMENT_STATUS_COMPLETED,
            page: page + 1,
          },
        });
```

Reply:
Thanks, good catch - I've also added it to all other async options props in this app as an improvement